### PR TITLE
Update links and description

### PR DESCRIPTION
**Pull request by Zalberda**
This stuff is out of date, its 2am, did what I could. Msg me on twitter at @alberdaZak if you have questions. Thanks.

### CODE REVIEW

File: docs/learn/uses-of-tezos/nft.md [hunk 1]
```diff
@@ -24,23 +24,22 @@ Recently, NFTs have been a powerful primitive demonstrating usefulness in the cr
...
 
-### Tools and Applications
-
-* \*\*\*\*[Truesy](https://www.truesy.com/) ****- a highly curated NFT marketplace
-* [Kalamint](https://kalamint.io/) - A community owned NFT marketplace on Tezos.
-* [OpenMinter](https://github.com/tqtezos/minter) ****and [Minter-SDK](https://github.com/tqtezos/minter-sdk) - A reusable dApp and SDK that allows anyone to create, collect, sell, or swap NFTs on Tezos.
-* \*\*\*\*[tzcolors](https://www.tzcolors.io)\*\*\*\*
-* \*\*\*\*[TezAuction](https://tez.auction/#/)\*\*\*\*
-* [CricTez](https://crictez.medium.com/introducing-crictez-21a284c60e94)
+### NFT Marketplaces
```
Comment (ilanbb1) [inline docs/learn/uses-of-tezos/nft.md:27]: Please remove descriptions for Objkt, Kalamint, and tzcolors

File: docs/learn/uses-of-tezos/nft.md [hunk 2]
```diff
@@ -24,23 +24,22 @@ Recently, NFTs have been a powerful primitive demonstrating usefulness in the cr
...
-* [OneOf](https://oneof.com/)
-* [RedBull Racing Honda](https://www.redbull.com/int-en/redbullracing/tezos-joins-as-official-blockchain-partner)
-* [McLaren Racing](https://www.mclaren.com/racing/partners/tezos/)
-* [Emergents TCG](https://www.emergents.gg/)
-* [Play with Brio ](https://playwithbrio.com/home)
-* [Tezos Profiles](https://tzprofiles.com/)
-* [Ipocamp](https://www.ipocamp.io/)
-
+* [OneOf](https://oneof.com/) - OneOf is a green NFT platform built for the music community to connect fans and collectors at all levels with their favorite artists.
+* [RedBull Racing Honda](https://redbullracingcollectibles.com/) - Built on Tezos, our blockchain partner, we’re bringing the world of Red Bull Racing closer to our fans with a series of amazing digital artifacts.
```
Comment (ilanbb1) [inline docs/learn/uses-of-tezos/nft.md:34]: Please edit name to "Red Bull Racing Honda", and please edit description "Bringing the world of Red Bull Racing closer to the fans with a series of amazing digital artifacts."

File: docs/learn/uses-of-tezos/nft.md [hunk 3]
```diff
@@ -24,23 +24,22 @@ Recently, NFTs have been a powerful primitive demonstrating usefulness in the cr
...
+* [McLaren Racing](https://mclarenracingcollective.com/) - The McLaren Racing Collective, powered by Tezos, is an innovative platform where fans and collectors can purchase McLaren Racing digital collectibles or non-fungible tokens (NFTs).
+* [Emergents TCG](https://emergentstcg.com/) - The trading card game where you own your cards
+* [Play with Brio ](https://playwithbrio.com/home) - Welcome to the first ever eSports skill based platform on the Tezos blockchain
```
Comment (ilanbb1) [inline docs/learn/uses-of-tezos/nft.md:37]: Please edit description to "The first ever eSports skill based platform on the Tezos blockchain"

File: docs/learn/uses-of-tezos/nft.md [hunk 4]
```diff
@@ -24,23 +24,22 @@ Recently, NFTs have been a powerful primitive demonstrating usefulness in the cr
...
+* [Tezos Profiles](https://tzprofiles.com/) - Tezos Profiles enables you to associate your online identity with your Tezos account.
```
Comment (ilanbb1) [inline docs/learn/uses-of-tezos/nft.md:38]: Please edit description to "Tezos profiles **enable** you to..."

File: docs/learn/uses-of-tezos/nft.md [hunk 5]
```diff
@@ -24,23 +24,22 @@ Recently, NFTs have been a powerful primitive demonstrating usefulness in the cr
...
+### Experimental/Future NFT Marketplaces
```
Comment (ilanbb1) [inline docs/learn/uses-of-tezos/nft.md:39]: Please remove this title and join the following three to the previous list

**Review by ilanbb1 — COMMENTED** [review]
Hi @Zalberda thank you for taking the time to put these edits together. I've made a few proofreading suggestions and would be happy to merge once reviewed. Thanks!

File: docs/learn/uses-of-tezos/nft.md (hunk 1 above)
Comment (Zalberda) [reply docs/learn/uses-of-tezos/nft.md:27]: Sorry I'm learning Github, idk how to edit the code.

**Pull request merged**